src/pk_core.h

```diff
@@ -62,4 +62,7 @@ Var varGetSubscript(PKVM* vm, Var on, Var key);
 // Set subscript [value] with the [key] (ie. on[key] = value).
 void varsetSubscript(PKVM* vm, Var on, Var key, Var value);
 
+// Validate a number is a byte
+bool isAByte(int64_t num);
+
```

Comment:
Even if we're using this as a function (instead of a macro), that shouldn't be public, but a static function. Remove this.